packages/frontend/src/components/molecules/dialog.molecule.tsx

```
@@ -0,0 +1,83 @@
+
+Dialog.Body = ({ children, className }: DialogBodyProps) => {
+  return <div className={cn('mt-4', className)}>{children}</div>;
+};
+
+Dialog.Footer = ({ children, className }: DialogFooterProps) => {
+  return <div className={cn('mt-6 flex items-center justify-end gap-3', className)}>{children}</div>;
+};
+
+Dialog.CloseButton = ({ onClose }: { onClose: () => void }) => {
+  return (
+    <button onClick={onClose} className="absolute right-4 top-4 rounded-lg p-1 text-gray-400 transition-colors hover:bg-gray-100 hover:text-gray-500 dark:hover:bg-gray-700 dark:hover:text-gray-300">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)